tests/manager/deleteCustomer/managerCanDeleteCustomer.spec.js

```diff
@@ -22,6 +35,12 @@ Test:
 4. Reload the page.
 5. Assert customer row is not present in the table. 
 */
+const customersListPage = new CustomersListPage(page); 
+const fullName = firstName + ' ' + lastName + ' ' + postCode;
```

Comment:
The `fullName` variable includes the postal code, which might not match the row's name attribute if it only contains the first and last name. Verify that the format of `fullName` matches the format used in the customer table for the row's name attribute.